### PR TITLE
enable user to configure rundeck running behind ssl proxy

### PR DESCRIPTION
**Pull request by jardleex**
Hi,

after some days here come the next PR.
It allows some settings required to make Rundeck work behind and SSL proxy like Apache or NGINX.

[Rundeck documentation](http://rundeck.org/docs/administration/configuring-ssl.html)

Best
 
Jard

### CODE REVIEW

File: templates/etc/rundeck/profile.j2 [hunk 1]
```diff
@@ -39,6 +39,9 @@ export RDECK_JVM="-Drdeck.config=/etc/rundeck \
     -Djava.security.auth.login.config=/etc/rundeck/jaas-loginmodule.conf \
     -Dloginmodule.name=RDpropertyfilelogin \
 {% endif %}
+{% if rundeck_jetty_connector_forwarded is defined %}
```
Comment (mrlesmithjr) [inline templates/etc/rundeck/profile.j2]: I would change this to be the following to ensure that this is only configured if `rundeck_jetty_connector_forwarded` is `true`:
```
{% if rundeck_jetty_connector_forwarded is defined and rundeck_jetty_connector_forwarded %}
    -Drundeck.jetty.connector.forwarded=true \
{% endif %}
```
Or...
```
{% if rundeck_jetty_connector_forwarded is defined %}
    -Drundeck.jetty.connector.forwarded={{ rundeck_jetty_connector_forwarded|lower }} \
{% endif %}
```

**Comment by jardleex** [comment]
Thanks for the good hint. I just added the first proposal to the branch.

**Comment by jardleex** [comment]
Hello @mrlesmithjr,

do you like to have some more adjustments here?
Can I assist you in any other way on this?

Best regards

Jard

**Comment by mrlesmithjr** [comment]
@jardleex Merged. Sorry for the delay. Thanks for the PR.

**Pull request merged**